### PR TITLE
Moe

### PR DESCRIPTION
**Pull request by J-Simkin**
Modified training parameters, added light regularization and normalized the output to the gatingMultiplier.
Now the network can be prone to overfitting and can take a couple attempts at training to get a good model but once one has been trained it can work nicely.
It does also require a lot of training data, I have been using a dataset of 4 movements with 25 samples each

### CODE REVIEW

**Review by Ike-G — APPROVED** [review]
lgtm!
